cLIMS/organization/migrations/0002_auto_20200529_1432.py

```diff
@@ -0,0 +1,110 @@
+            model_name='experiment',
+            name='document',
+            field=models.ForeignKey(blank=True, help_text='Documents that provide additional information (not data file).', null=True, on_delete=django.db.models.deletion.SET_NULL, to='wetLab.Document'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='experiment_biosample',
+            field=models.ForeignKey(help_text='Starting biological material.', on_delete=django.db.models.deletion.CASCADE, related_name='expBio', to='wetLab.Biosample'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='experiment_enzyme',
+            field=models.ForeignKey(blank=True, help_text='The enzyme used for digestion of the DNA.', null=True, on_delete=django.db.models.deletion.CASCADE, related_name='expEnz', to='wetLab.Enzyme'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='imageObjects',
+            field=models.ManyToManyField(blank=True, help_text='Any additional image related to this experiment.', related_name='expImg', to='dryLab.ImageObjects'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='project',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='expProject', to='organization.Project'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='protocol',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='expPro', to='wetLab.Protocol'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='references',
+            field=models.ForeignKey(blank=True, help_text='The publications that provide more information about the object.', null=True, on_delete=django.db.models.deletion.SET_NULL, to='organization.Publication'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='type',
+            field=models.ForeignKey(help_text='JsonObjField', on_delete=django.db.models.deletion.CASCADE, related_name='expType', to='organization.JsonObjField'),
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='variation',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='expProVar', to='wetLab.Protocol', verbose_name='protocol_variations'),
+        ),
+        migrations.AddField(
+            model_name='award',
+            name='award_exp',
+            field=models.ManyToManyField(related_name='awardExp', to='organization.Experiment'),
+        ),
+        migrations.AddField(
+            model_name='award',
+            name='award_project',
+            field=models.ManyToManyField(related_name='awardPro', to='organization.Project'),
+        ),
+    ]
```

Comment:
question about the file below - DCIC-Metadata_entry_form_FEB2018.xlsx - is it a new form ? do we keep forms as a part of the github repo ? - If that's the case - than ok

Reply:
@Snehal1894 how about this one ? are we keeping those as part of the github repository ?

Reply:
No, we need to update this excel sheet. The old sheet has existing entries ( that was the main issue, because of which the extra entries were seen). We need to update this on the server.

Reply:
ok, so do we need to remove it from the repository ?

is this automatically generated by cLIMS or is it some form that one get from DCIC ?

Reply:
Generated by cLIMS. Nothing directly related to DCIC. Automatically considers our updates in code